src/tools/sdk/archive/index.md

```diff
@@ -40,7 +41,7 @@ production use.
 
 {% include_relative _archives_table.html channel="dev" %}
 
-## Direct download URLs
+## Download URLs
```

Comment:
We should update the example URLs, either now or in a future PR.

Reply:
I've made some updates to the example URLs. We can make more once 2.0 final comes out.